Atomic/scan.py

```diff
@@ -170,11 +170,12 @@ def gen_images():
         def gen_containers():
             slist = []
             for con in self.get_containers():
-                if con['Status'].upper() != 'DEAD':
-                    con['input'] = con['Id']
-                else:
-                    raise ValueError("The container ID {} is Dead and therefor cannot be scanned.".format(con['Id'][:12]))
-                slist.append(con)
+                if 'Status' in con:
```

Comment:
Rather than having the IF conditional to see if Status exists in con and then getting the value, .get combines both.  If Status does not exist, then the value is None.